lwilcox900@wilcotron-nas:/volume1/docker/selling-options-lab$ sudo docker compose logs selling-options-lab-app
WARN[0000] Found multiple config files with supported names: /volume1/docker/selling-options-lab/compose.yaml, /volume1/docker/selling-options-lab/docker-compose.yml
WARN[0000] Using /volume1/docker/selling-options-lab/compose.yaml
WARN[0000] Found multiple config files with supported names: /volume1/docker/selling-options-lab/compose.yaml, /volume1/docker/selling-options-lab/docker-compose.yml
WARN[0000] Using /volume1/docker/selling-options-lab/compose.yaml
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
selling-options-lab-app  | Traceback (most recent call last):
selling-options-lab-app  |   File "/app/app.py", line 14, in <module>
selling-options-lab-app  |     import yfinance as yf
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/__init__.py", line 23, in <module>
selling-options-lab-app  |     from .ticker import Ticker
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/yfinance/ticker.py", line 25, in <module>
selling-options-lab-app  |     import pandas as _pd
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/__init__.py", line 22, in <module>
selling-options-lab-app  |     from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
selling-options-lab-app  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/__init__.py", line 25, in <module>
selling-options-lab-app  |     from pandas.compat.numpy import (
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/compat/numpy/__init__.py", line 4, in <module>
selling-options-lab-app  |     from pandas.util.version import Version
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/__init__.py", line 2, in <module>
selling-options-lab-app  |     from pandas.util._decorators import (  # noqa:F401
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/util/_decorators.py", line 14, in <module>
selling-options-lab-app  |     from pandas._libs.properties import cache_readonly
selling-options-lab-app  |   File "/usr/local/lib/python3.11/site-packages/pandas/_libs/__init__.py", line 13, in <module>
selling-options-lab-app  |     from pandas._libs.interval import Interval
selling-options-lab-app  |   File "pandas/_libs/interval.pyx", line 1, in init pandas._libs.interval
selling-options-lab-app  | ValueError: numpy.dtype size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject
lwilcox900@wilcotron-nas:/volume1/docker/selling-options-lab$